### ISSUE TITLE
Docker images running EOL version of Alpine Linux

### ISSUE THREAD

**Issue by ImperialXT** [hashicorp/packer#8674]
#### Overview of the Issue
The official packer docker images published at https://hub.docker.com/r/hashicorp/packer/ are being built with an old and EOL version of Alpine Linux (3.7.0).

Others have raised this issue at the repo that docker hub indicates is the source of the image. With no response: https://github.com/hashicorp/docker-hub-images/issues/83

#### Reproduction Steps

```bash
$ docker pull hashicorp/packer:light
light: Pulling from hashicorp/packer
ff3a5c916c92: Pull complete
994b12bf22d2: Pull complete
4d35fed13094: Pull complete
1a3ec813d65f: Pull complete
87cc5ddb914b: Pull complete
68f6895e03c3: Pull complete
be6e4468e037: Pull complete
Digest: sha256:20f3f783025f293f44f44c8eb6ca0f6a1809b76e1836d64444de3b4300ccd97b
Status: Downloaded newer image for hashicorp/packer:light
docker.io/hashicorp/packer:light
$ docker run --entrypoint /bin/bash -it hashicorp/packer:light
bash-4.4# cat /etc/alpine-release
3.7.0
bash-4.4# exit
exit

```

**Comment by SwampDragons** [comment]
Thanks for bringing this to my attention. I'll take a look before the next Packer release.

**Comment by SwampDragons** [comment]
This has been fixed for the 1.5.2 docker release: 

```
docker run --entrypoint /bin/bash -it hashicorp/packer:light
bash-5.0# cat /etc/alpine-release
3.11.3
bash-5.0# exit
```

**Comment by SwampDragons** [comment]
I realized I didn't fix it for the "full" image; rebuilding it now.

**Comment by ImperialXT** [comment]
I notice you haven't pinned to a particular version of alpine and are instead still using `latest` my remaining concern is that if it remains on `latest` it'll be a problem again in a few months or similar.

**Comment by SwampDragons** [comment]
I agree. We have a little deploy script we use to build and upload these files -- I'm adding a docker pull of the base images to make sure the images and tags are updated with each update to the dockerfile.

I'll close this issue after I update the script.

**Comment by SwampDragons** [comment]
This is fixed now for the v1.5.2 release, and I've made a change to the deploy script we use to force updates to keep it on the actual latest container.
